### ISSUE TITLE
Implement launcher code

### ISSUE THREAD

**Issue by ksaric** [input-output-hk/cardano-shell#41]
After #28 we can start moving the launcher code into `cardno-shell`.
The code for running it here - https://github.com/input-output-hk/cardano-sl/blob/develop/tools/src/launcher/Main.hs
The supporting code here - https://github.com/input-output-hk/cardano-sl/tree/develop/lib/src/Pos/Launcher

**Comment by ksaric** [comment]
This is way too broad. As discussed in the meeting today, we can split this up. Closing.
